templates/gallery/partials/landing-images.snip.html

```diff
@@ -0,0 +1,42 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#landing-images}}
+  <figure class="gallery-landing-image-wrap-{{index}}">
+    <a href="./gallery-inner.amp.html" class="gallery-landing-image-link"></a>
```

Comment:
i think the anchor should wrap the entire figure.

Reply:
This was done deliberately, so that the target for clicking an image is slightly smaller than the image itself. This was necessary because some of the images overlap.

Reply:
I see , thanks for the explanation!